compiler/semstmts.nim

```diff
@@ -2035,6 +2035,8 @@ proc evalInclude(c: PContext, n: PNode): PNode =
   for i in 0 ..< sonsLen(n):
     var imp: PNode
     let it = n.sons[i]
+    if it.kind == nkInfix and it.len == 3 and it[0].ident.s == "as":
+      localError(c.config, it.info, "Cannot use 'as' in 'include'.")
```

Comment:
Instead of blacklisting the `as` operator I thought more of a whitelisting solution for infix operators that are allowed.